src/styles/Rating.css

```diff
@@ -0,0 +1,39 @@
+.rate {
+  float: left;
```

Comment:
any reason to use float, instead of flex

Reply:
I used Grid in this